add_item.php

```diff
@@ -39,13 +40,13 @@
             <label for="incubation-range">Incubation range</label>
             <input type="text" name="incubation-range" id="incubation-range" required>
             <label for="symptoms">Symptoms</label>
-            <input type="text" name="symptoms" id="symptoms">
+            <input type="text" name="symptoms" id="symptoms" required>
             <label for="severity">Severity</label>
-            <input type="text" name="severity" id="severity">
+            <input type="text" name="severity" id="severity" required>
             <label for="avg-annual-incidence">Average annual incidence</label>
-            <input type="number" name="avg-annual-incidence" id="avg-annual-incidence">
+            <input type="number" name="avg-annual-incidence" id="avg-annual-incidence" required>
             <label for="disease-img">Image</label>
-            <input type="file" name="disease-img" id="disease-img">
+            <input type="file" name="disease-img" id="disease-img" required>
```

Comment:
Would remove `required`, check if this is empty and in that case not update that field in the db

OR

Populate this field with something, check to make sure the value is what you provided and not update that field in the db